src/rt/aaA.d

```diff
@@ -480,7 +545,7 @@ inout(ArrayRet_t) _aaKeys(inout AA aa, in size_t keysize, const TypeInfo tiKeyAr
     if (!len)
         return null;
 
-    void[] res = _d_newarrayU(tiKeyArray, len);
+    void* res = _d_newarrayU(tiKeyArray, len).ptr;
```

Comment:
This change is unrelated, but the current version causes a range error when building druntime without -release.
The return value of _d_newarrayU is confusing, as it actually returns the length for `T[]`, not `void[]`. It should rather just return `void*`